pkg/services/publicdashboards/publicdashboard.go

```diff
@@ -17,7 +17,7 @@ type Service interface {
 	FindPublicDashboardAndDashboardByAccessToken(ctx context.Context, accessToken string) (*PublicDashboard, *models.Dashboard, error)
 	FindByDashboardUid(ctx context.Context, orgId int64, dashboardUid string) (*PublicDashboard, error)
 	FindAnnotations(ctx context.Context, reqDTO AnnotationsQueryDTO, accessToken string) ([]AnnotationEvent, error)
-	FindDashboard(ctx context.Context, dashboardUid string, orgId int64) (*models.Dashboard, error)
```

Comment:
why'd these get swapped?

Reply:
Just an unwritten standard- we take in context, then orgId in all other methods

Reply:
Because we were using this order in other functions and also I like to keep on the left the broader parameters